staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -253,7 +253,7 @@ type VirtualMachineInstanceStatus struct {
 	// +optional
 	TopologyHints *TopologyHints `json:"topologyHints,omitempty"`
 
-	//VirtualMachineRevisionName is used to get the vm revision of the vmi when doing
+	// VirtualMachineRevisionName is used to get the vm revision of the vmi when doing
```

Comment:
Thanks but in the future I'd rather see unrelated cleanups like this in another PR.